### PR TITLE
Sam/privacy link

### PR DESCRIPTION
**Pull request by smarvin**
**Things To Remember:** 

1. Be sure to merge into the right branch. The choices are:
* **`master`** if your PR runs on the current version of Tableau.
* **`dev-2021.2`**  
2. You should update at least one of the following:
* The changelog of the project you modified.
* The root `README.md` if the PR is a release of the SDK, Connector Packager, or TDVT.  
=====

### CODE REVIEW

**Comment by smarvin** [comment]
@rosswbrown , looks like there's some older stuff here. Not sure what you need me to do. It looks messy, so I'm happy to back out any requests and start again if that's helpful.

File: docs/capabilities.md [hunk 1]
```diff
@@ -107,7 +107,9 @@ CAP_QUERY_TOP_PERCENT | Set to 'yes' if TOP supports percent | no | **<span styl
 CAP_QUERY_TOP_SAMPLE | Set to 'yes' if TOP supports sampling rows | no | **<span style="color:red">?</span>**  
 CAP_QUERY_TOP_SAMPLE_FAST | Set to 'yes' if TOP with sampling is faster than TOP | no | no 
 CAP_QUERY_TOP_SAMPLE_PERCENT | Set to 'yes' if TOP supports sampling by percentages | no | **<span style="color:red">?</span>**  
-CAP_QUERY_USE_DOMAIN_RANGES_OPTIMIZATION | Set to 'no' if domain ranges cannot be used to optimize set functions. | yes | yes
+CAP_QUERY_USE_DOMAIN_EXCEPT_OPTIMIZATION | When the number of selected items in a filter is greater than half the total number of items in the domain, save the excluded items instead of the included items to save space.  Available in Tableau 2022.2 and newer. | yes | yes 
+CAP_QUERY_USE_DOMAIN_RANGES_OPTIMIZATION | Set to 'no' if domain ranges cannot be used to optimize set functions. | yes | yes 
+CAP_QUERY_USE_DOMAIN_RANGES_OPTIMIZATION_STRINGS | Set to ‘yes’ if domain ranges should be used to optimize set functions for string data. Available in Tableau 2022.2 and newer. | no | no
```
Comment (pvanderknyff) [inline docs/capabilities.md:112]: Are these changes intentional? Seems like you got one of Ross's commits in there somehow.

**Comment by rosswbrown** [comment]
> @rosswbrown , looks like there's some older stuff here. Not sure what you need me to do. It looks messy, so I'm happy to back out any requests and start again if that's helpful.

Yes, please ensure your feature/task branch is up to date and re-push so we don't see anything unexpected.

File: docs/capabilities.md (hunk 1 above)
Comment (smarvin) [reply docs/capabilities.md:112]: Yes, these changes were two new capabilities I was asked to add.

**Review by rosswbrown — CHANGES_REQUESTED** [review]
Ensure your feature branch is up to date with current gh-pages. Those were added in [972](https://github.com/tableau/connector-plugin-sdk/pull/972) and merged in as a part of the 2022.2 [release](https://github.com/tableau/connector-plugin-sdk/pull/975)
